### DISCUSSION TITLE
saf-training-current/courses/profile-dev-test/24

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#55] [Ideas]
# saf-training-current/courses/profile-dev-test/24

Preparing the Profile Before Running Delta Before running Delta, it's beneficial to format the profile to match the format Delta will use. This minimizes changes to only those n...

https://mitre.github.io/saf-training-current/courses/profile-dev-test/24.html

**Comment by wdower** [comment]
note -- probably want to give a full example of the prep step including the flags for the XCCDF file etc
